docs/user/howto/pull.md

```diff
@@ -0,0 +1,56 @@
+# Pulling charts to the local machine
+
+Sure, installing charts from an url or repo is cool, but sometimes you need to modify those a bit locally or just have a look at what the chart is actually doing.
```

Comment:
```suggestion
Sure, installing charts from an url or repo is cool, but sometimes you need to modify those locally a bit, or just have a look at what the chart is actually doing.
```

(if not, to me it sounds that you need a bit of locality, not that you are going to modify a bit, locally).

Reply:
Also I would like to see a more technical "lingo" here like
`- Sometimes there is need to modfify charts locally.....`

Reply:
i consider this fixed, with https://github.com/rancher-sandbox/hypper/pull/111/commits/83b60c5855566b9a880a2030a35bf8e8f6cb241e.